pkg/reconciler/channel/channel.go

```diff
@@ -250,10 +252,11 @@ func (c *Reconciler) syncSubscribers(ctx context.Context, channel *v1alpha1.Chan
 		}
 		c.Recorder.Eventf(channel, corev1.EventTypeNormal, "CreatedSubscriber", "Created Subscriber %q", genName)
 
-		channel.Status.Subscribers = append(channel.Status.Subscribers, eventingduck.SubscriberStatus{
+		channel.Status.SubscribableStatus.Subscribers = append(channel.Status.SubscribableStatus.Subscribers, eventingduck.SubscriberStatus{
 			UID:                s.UID,
 			ObservedGeneration: s.Generation,
-			// TODO: do I need the other fields?
+			// TODO read the condition from PullSubscription, and add a message if it's not ready.
```

Comment:
read the condition from PullSubscription, and add a message if it's not ready.

Reply:
done!